### DISCUSSION TITLE
Sensors are not selectable to Configure Gas Consumption in Energy dashboard settings

### DISCUSSION THREAD

**Discussion by nicolewells** [HandyHat/ha-hildebrandglow-dcc#263] [Q&A] [answered]
Hi,

Can you think of a reason why the various gas consumption sensors are not available in the dropdown when trying to setup the energy dashboard via the UI for consumption? 

The rate allows the selection of the Gas rate - it's just the consumption dropdown.
I've been able to setup the Electricity without issue.
All sensor entities appear correctly within the developer view.

Thanks for any help and many thanks for your work.

Regards
Nicole

**Comment by jc12col** [comment]
Same here !

**Comment by HandyHat** [comment, marked as answer]
This seems to be a bug in Home Assistant Core - see #264

**Comment by nicolewells** [comment]
Thanks,

Yes, it seems so. 

FYI - I've managed to rectify my situation by editing the home assistant energy.py file manual to add the gas consumption entity manually and this has worked for me.

Thanks for your wonderful API btw.

And I second the enhancement to retrieve the available history consumption information if possible. 

Kind Regards

Nicole Wells
